unime/src/routes/(app)/scan/+page@.svelte

```diff
@@ -296,10 +297,10 @@
       <div class="flex grow flex-col" class:invisible={!scanning}>
         <div class="bg-white p-5 dark:bg-dark">
           <p class="text-3xl font-semibold text-slate-700 dark:text-grey">
-            Scan a <span class="text-primary">QR Code</span>
+            {$LL.SCAN.TITLE_1()} <span class="text-primary"> {$LL.SCAN.TITLE_2()}</span>
```

Comment:
We still need the first space before the span

```
{$LL.SCAN.TITLE_1()} <span class="text-primary">{$LL.SCAN.TITLE_2()}</span>
                    ^
```

Otherwise it will look like this: 
<img width="359" alt="Screenshot 2024-02-16 at 06 44 38" src="https://github.com/impierce/identity-wallet/assets/45594838/68420256-83fd-4717-b474-8c0f75a4c7de">